airbyte-server/src/test/java/io/airbyte/server/handlers/ConnectionsHandlerTest.java

```diff
@@ -612,6 +619,321 @@ void testEnumConversion() {
       assertTrue(Enums.isCompatible(NamespaceDefinitionType.class, io.airbyte.config.JobSyncConfig.NamespaceDefinitionType.class));
     }
 
+    @Test
+    void testNoDiff() {
+      final AirbyteStreamConfiguration streamConfiguration1 = getStreamConfiguration(List.of("cursor1"), List.of(List.of("pk1")), SyncMode.INCREMENTAL,
+          DestinationSyncMode.APPEND_DEDUP);
+
+
+      final AirbyteStreamConfiguration streamConfiguration2 = getStreamConfiguration(List.of("cursor2"), List.of(List.of("pk2")),
+          SyncMode.FULL_REFRESH,
+          DestinationSyncMode.OVERWRITE);
```

Comment:
nit: for all of these lines where you call `getStreamConfiguration()`, put each argument on a new line. This way it is easier to compare one call with another and see what is different, which is a bit hard right now due to the differences in formatting

Reply:
👍